lib/settings.js

```diff
@@ -41,7 +41,11 @@ function readFrom(envVars) {
         STARTUP_REQUIRED: false,
         HOOK_SHUTDOWN: false,
         API_REQUEST_RETRY_ATTEMPTS: 3,
-        API_REQUEST_RETRY_DELAY: 100
+        API_REQUEST_RETRY_DELAY: 100,
+        DATA_RATE_LIMIT: 100,
+        ERROR_RATE_LIMIT: 20,
+        SNAPSHOT_RATE_LIMIT: 20,
+        RATE_INTERVAL: 1000 // One second
```

Comment:
Due to implementation details of the library we use it's much better to use 10 requests every 100 ms than 100 requests every 1000 ms. Could you please adjust values of the constants accordingly?